### PR TITLE
[0.11] Add datadog and graphite reporter

### PR DESCRIPTION
**Pull request by aquamatthias**

### CODE REVIEW

File: src/main/scala/mesosphere/marathon/metrics/MetricsReporterService.scala [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  }
+
+  def startUp() {
+    this.graphite = config.graphite.get.map(startGraphiteReporter)
+    this.datadog = config.dataDog.get.map(startDatadog)
+  }
+
+  def shutDown() {
+    graphite.foreach(_.stop)
+    datadog.foreach(_.stop())
```
Comment (gkleiman) [inline src/main/scala/mesosphere/marathon/metrics/MetricsReporterService.scala]: Why do you have the parens here and not in the previous line?

**Comment by gkleiman** [comment]
Looks good to me, but I think we should document the reporters.

**Comment by gkleiman** [comment]
I just noticed that this is a backport to 0.11.1.

I think my comments only make sense for master. If you agree with them, then I'll just merge this and create an issue.

File: src/main/scala/mesosphere/marathon/metrics/MetricsReporterService.scala (hunk 1 above)
Comment (aquamatthias) [inline src/main/scala/mesosphere/marathon/metrics/MetricsReporterService.scala]: No reason and should be consistent.

**Comment by aquamatthias** [comment]
I agree.

**Pull request merged**